### PR TITLE
Cluster Loader initial concurrency

### PR DESCRIPTION
**Pull request by sjug**
This PR adds concurrency to Cluster Loader, as described [here](https://trello.com/c/VgFjwkQs]).

### CODE REVIEW

**Comment by sjug** [comment]
@vbatts Could you give me a review please?

**Comment by jmencak** [comment]
Has the problem with the template creation in the last namespace been fixed?

**Comment by vbatts** [comment]
@sjug it's definitely less nested loops.

File: test/extended/cluster/cl.go [hunk 1]
```diff
@@ -56,77 +59,18 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
...
-			e2e.Failf("Invalid config file.\nFile: %v", project)
+		projects := ConfigContext.ClusterLoader.Projects
+		if projects == nil {
+			e2e.Failf("Invalid config file.\nFile: %v", projects)
 		}
-
-		var namespaces []string
-		//totalPods := 0 // Keep track of how many pods for stepping
-		// TODO sjug: add concurrency
+		sync := ConfigContext.ClusterLoader.Sync
```
Comment (vbatts) [inline test/extended/cluster/cl.go]: is this intending to initialize something? or just shorten the variable name used? Cause `sync` var name is now shadowing the imported package `sync` ...

**Comment by sttts** [comment]
Is it only me who cannot ready the linked trello card? Grey screen.

**Comment by sttts** [comment]
@sjug can you split the PR into one commit just moving code into the funcs and one changing the logic? In the current way this is very hard to review.

File: test/extended/cluster/cl.go (hunk 1 above)
Comment (sjug) [reply test/extended/cluster/cl.go]: Just to shorten the variable name

**Pull request closed without merging**